File: app/src/main/java/org/docheinstein/minimote/ui/settings/SettingsFragment.java
package org.docheinstein.minimote.ui.settings;

import android.os.Bundle;

import org.docheinstein.minimote.R;
import org.docheinstein.minimote.ui.base.MinimotePreferenceFragment;

public class SettingsFragment extends MinimotePreferenceFragment {
    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.preferences, rootKey);
    }
}

File: app/src/main/java/org/docheinstein/minimote/ui/settings/SettingsFragment.kt
package org.docheinstein.minimote.ui.settings

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import org.docheinstein.minimote.R

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.preferences, rootKey)
    }
}